src/main/resources/application.yml

```diff
@@ -1,2 +1,18 @@
 server:
   port: 8081
+spring:
+  datasource:
+    driver-class-name: org.postgresql.Driver
+    username: admin_wst
+    password: wst!#4pas@*
+    url: jdbc:postgresql://localhost:5432/db_wst
+    platform: postgres
+    initialization-mode: always
+  jpa:
+    show-sql: true
+    hibernate.ddl-auto: none
+    database-platform: org.hibernate.dialect.PostgreSQL94Dialect
+    properties:
+      hibernate:
+        temp:
+          use_jdbc_metadata_defaults: false
```

Comment:
What is this property? where did you get this example?

Reply:
https://o7planning.org/ru/11897/spring-boot-and-spring-data-jpa-tutorial

# Fix Postgres JPA Error:
# Method org.postgresql.jdbc.PgConnection.createClob() is not yet implemented.
spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults=false